apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -171,7 +176,9 @@ function AssessmentQuestionsTable({
                           output: question.sync_warnings,
                         })
                       : ''}
-                  ${question.display_name}
+                  ${idsEqual(course.id, question.course_id)
+                    ? question.qid
+                    : html`@${question.course_sharing_name}/${question.qid}`}
```

Comment:
Interpolating a string into the parent `html` tagged template literal will already do this.

```suggestion
                    : `@${question.course_sharing_name}/${question.qid}`}
```